content/doc/book/pipeline/index.adoc

```diff
@@ -120,6 +143,76 @@ modeled in Jenkins Pipeline:
 
 image::pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]
 
+[[jenkinsfile]]
+== The Jenkinsfile
+
+A `Jenkinsfile` is a text file that contains the definition of a
```

Comment:
Should Jenkinsfile here emulate the revised form? (<<Jenkinsfile,`Jenkinsfile`>>)